### PR TITLE
Use wp cli to set development mode true

### PR DESCRIPTION
**Pull request by mikeritter**
Adds instructions to set development mode to true using wp cli in the terminal

**Description**

**Screenshots**

**Testing Instructions**

### CODE REVIEW

**Review by juanfra — APPROVED** [review]
Thanks for the PR!

✅ LGTM

File: README.md [hunk 1]
```diff
@@ -33,6 +33,8 @@ To get started with development:
 
 Also, consider enabling [development mode](https://make.wordpress.org/core/2023/07/14/configuring-development-mode-in-6-3/) with `define( 'WP_DEVELOPMENT_MODE', 'theme' );` in your `wp-config.php`. This will help minimize caching of `theme.json` while you're developing.
 
+To enable development mode using [WP CLI](https://developer.wordpress.org/cli/commands/config/set/) use the following command in your terminal inside your installation directory `wp config set 'WP_DEVELOPMENT_MODE' true --raw`.
```
Comment (juanfra) [inline README.md]: Should it be `theme` instead of `true`?

Comment (mikeritter) [reply README.md]: thank you

**Comment by huzaifaalmesbah** [comment]
Great! This is very helpful and looks good.

File: README.md [hunk 2]
```diff
@@ -33,6 +33,8 @@ To get started with development:
 
 Also, consider enabling [development mode](https://make.wordpress.org/core/2023/07/14/configuring-development-mode-in-6-3/) with `define( 'WP_DEVELOPMENT_MODE', 'theme' );` in your `wp-config.php`. This will help minimize caching of `theme.json` while you're developing.
 
+To enable development mode using [WP CLI](https://developer.wordpress.org/cli/commands/config/set/) use the following command in your terminal inside your installation directory `wp config set 'WP_DEVELOPMENT_MODE' theme --raw`.
```
Comment (juanfra) [inline README.md]:
```suggestion
To enable development mode using [WP CLI](https://developer.wordpress.org/cli/commands/config/set/) use the following command in your terminal inside your installation directory `wp config set 'WP_DEVELOPMENT_MODE' theme`.
```

**Review by juanfra — CHANGES_REQUESTED** [review]
Thanks for the changes, we should also remove the `--raw` flag so that it places the value in a quoted string.

**Review by mikeritter — COMMENTED** [review]
cleaned up code example and corrected typo

**Review by juanfra — APPROVED** [review]
Thank you @mikeritter 🚀

**Pull request merged**